replicate-to-sql-with-generic-events/scripts/delete-data.js

```diff
@@ -0,0 +1,37 @@
+const { Kuzzle, WebSocket } = require('kuzzle-sdk');
+
+const kuzzle = new Kuzzle(new WebSocket('localhost'));
+
+const indexName = 'nyc-open-data';
+const collectionName = 'yellow-taxi';
+
+async function searchData() {
```

Comment:
There is a lot of overhead introduced by this function since you are awaiting the promise, then wrapping the result in another promise before returning it.  
IMHO, this call could be put directly in the script